conn.go

```diff
@@ -507,6 +513,17 @@ func (c *Conn) WriteMessages(msgs ...Message) (int, error) {
 		return 0, nil
 	}
 
+	// users may believe they can set the Topic and/or Partition
+	// on the kafka message.
```

Comment:
Let’s add this to te WriteMessages documentation so we leave no room for interpretation here.